### ISSUE TITLE
[BUG] OpenApi 2 client generator refparameters not separated by slash

### ISSUE THREAD

**Issue by karismann** [swagger-api/swagger-parser#1067]
##### Description

In an openAPI 2.0 declaration, if you have two ref parameters not separated by "/" , the client generator send the error : 

> -attribute paths.'/v1/users/{firstparam}-{secondparam}'. Declared path parameter firstparam}-{secondparam needs to be defined as a path parameter in path or operation level

##### swagger parser version

2.0.11-OpenAPITools.org-1

##### OpenAPI declaration file content or url

```json
"paths" : {
    "/v1/users/{firstparam}-{matricule}" : {
      "get" : {
        "description" : "",
        "operationId" : "xxx",
        "produces" : [ "application/json" ],
        "parameters" : [ {
          "name" : "firstparam",
          "in" : "path",
          "required" : true,
          "type" : "string"
        }, {
          "name" : "secondparam",
          "in" : "path",
          "required" : true,
          "type" : "string"
        } ],
```

**Comment by macjohnny** [comment]
@karismann shouldn't the path be `/v1/users/{firstparam}-{secondparam}`?

**Comment by karismann** [comment]
> @karismann shouldn't the path be `/v1/users/{firstparam}-{secondparam}`?

you're right, that's a bad copy-paste, i fix my description, but the problem is still here  ;-)

**Comment by mthebridge** [comment]
I have also hit this (on `OpenAPI-tools-2.0.8-2`), and it seems to be a regression cuased by #928 - the check there that path parameters are defined assumes that the path parametrs form the whole of a URl segment.
